frontend/apps/reader/modules/readersearch.lua

```diff
@@ -79,6 +80,12 @@ SRELL_ERROR_CODES[110] = _("No preceding expression in repetition.")
 SRELL_ERROR_CODES[111] = _("Expression too complex, some hits will not be shown.")
 SRELL_ERROR_CODES[666] = _("Expression may lead to an extremely long search time.")
 
+function ReaderSearch:registerKeyEvents()
+    if Device:hasKeyboard() then
+        self.key_events.ShowFulltextSearchInput = { { Device.input.group.AlphaNumeric }, args = true }
```

Comment:
```suggestion
        self.key_events.ShowFulltextSearchInput = { { "F" }, args = true }
```

Reply:
We don't use `find` anywhere. What we do use is the somewhat awkward `Fulltext search`, which should perhaps be something like `Search in text`.

Reply:
And with this change, do you get F or S added in the input field ?

Reply:
> And with this change, do you get F or S added in the input field ?

not with the `args = true`, without it, it does bleed through, incidentally the File search in FM bleeds through as well, you always start your search with a capital `S`

Reply:
> incidentally

Maybe it's just a last search string? Why don't you want to understand what you are doing?

`args` is passed to the event
https://github.com/koreader/koreader/blob/7b9c65ee1ed43020473dbcaf321ea85bee34eef5/frontend/ui/widget/container/inputcontainer.lua#L235
Text search doesn't get any args
https://github.com/koreader/koreader/blob/7b9c65ee1ed43020473dbcaf321ea85bee34eef5/frontend/apps/reader/modules/readersearch.lua#L208
https://github.com/koreader/koreader/blob/7b9c65ee1ed43020473dbcaf321ea85bee34eef5/frontend/apps/reader/modules/readersearch.lua#L217
File search does
https://github.com/koreader/koreader/blob/7b9c65ee1ed43020473dbcaf321ea85bee34eef5/frontend/apps/filemanager/filemanagerfilesearcher.lua#L26
https://github.com/koreader/koreader/blob/7b9c65ee1ed43020473dbcaf321ea85bee34eef5/frontend/apps/filemanager/filemanagerfilesearcher.lua#L31

Reply:
> Maybe it's just a last search string? 

is not, it does bleed through.

> Why don't you want to understand what you are doing?

I know what is happening there, I just haven't changed it (for filesearcher), I was just mentioning it.

Reply:
> > And with this change, do you get F or S added in the input field ?
> 
> not with the `args = true`, without it, it does bleed through, incidentally the File search in FM bleeds through as well, you always start your search with a capital `S`

I lied (involuntarily though). why do things work one way when I test them the first time around and then act differently the next time around. It **does** bleed through even with `args=true` which come to think of, was only there to eliminate the previous search so touch'e (on myself, sorry @hius07)